feature-libs/organization/administration/components/shared/list/list.service.ts

```diff
@@ -158,4 +158,24 @@ export abstract class ListService<T, P = PaginationModel> {
     pagination: PaginationModel,
     ...args: any
   ): Observable<EntitiesModel<T> | undefined>;
+
+  /**
+   * If this method returns true, hyperlink will be shown to add new user.
+   * If this method returns false, button will be displayed.
+   */
+  showLink(): boolean {
+    return true;
+  }
+
+  /**
+   * This method will be called when the button to create new user is clicked.
```

Comment:
Let's replace the word `user` with `item` in this code comment. This service is generic - used not only for Users, but also for Budgets, CostCenters, etc.

Reply:
done